### PR TITLE
Let pybind11 cast bool sequences

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

File: _pyntcore/src/py2value.cpp [hunk 1]
```diff
@@ -81,11 +81,7 @@ std::shared_ptr<nt::NetworkTableValue> py2ntvalue(py::handle h) {
   // check the first item
   auto i1 = seq[0];
   if (py::isinstance<py::bool_>(i1)) {
-    std::vector<int> v;
-    v.reserve(seq.size());
-    for (size_t i = 0; i < seq.size(); i++) {
-      v.push_back(seq[i].cast<int>());
-    }
+    auto v = h.cast<std::vector<int>>();
```
Comment (ArchdukeTim) [inline _pyntcore/src/py2value.cpp]: why not follow the form of the rest and return the cast directly?

Comment (auscompgeek) [reply _pyntcore/src/py2value.cpp]: I don't understand the question. This _is_ written like the rest.

Comment (ArchdukeTim) [reply _pyntcore/src/py2value.cpp]: You create a variable, the others don't have a variable

Comment (ArchdukeTim) [reply _pyntcore/src/py2value.cpp]: oh I guess this lower section is different from above. carry on

**Pull request merged**